app/src/main/java/com/sun/basic_japanese/data/model/Alphabet.kt

```diff
@@ -0,0 +1,24 @@
+package com.sun.basic_japanese.data.model
+
+import android.database.Cursor
+import com.sun.basic_japanese.data.source.local.AppDatabase
+
+data class Alphabet(
```

Comment:
```suggestion
data class Alphabet(
```
nếu nhiều params trong constructor nên khai triển builder pattern để đảm báo clean code với tạo ra 1 immutable class.
https://gpcoder.com/4434-huong-dan-java-design-pattern-builder/